lib/Zonemaster/Backend/Config.pm

```diff
@@ -59,7 +65,16 @@ file.
         }
     );
 
-Throws an exception if the given configuration file contains errors.
+The configuration is interpreted according to the
+L<configuration format specification|https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md>.
+
+Returns a new Z::B::Config instance with its properties set to values according
```

Comment:
I think it would be better to avoid abbreviation in documentation and explicitly write `Zonemaster::Backend::Config`.

Reply:
I agree completely. I'll make an update.

Reply:
Mee to. I think we should not use such a breviation anywhere, not even in issue or PR descriptions.